nvflare/private/fed/server/server_engine.py

```diff
@@ -439,7 +444,9 @@ def new_context(self) -> FLContext:
         if self.run_manager:
             return self.run_manager.new_context()
         else:
-            return FLContext()
+            # return FLContext()
+            return FLContextManager(engine=self, identity_name=self.server.project_name, run_num="",
+                                    public_stickers={}, private_stickers={}).new_context()
```

Comment:
what's the difference between this and the previous method of creating a FLContext?

Reply:
FLContextManager.new_context() will create default properties instead of empty one.